content/run-nym-nodes/troubleshooting.md

```diff
@@ -0,0 +1,283 @@
+---
+title: "Troubleshooting FAQ"
+weight: 30
```

Comment:
Hey if you change this to 40 it'll drop to the bottom, and I'll merge the pull :)